spec/features/feature_tests_spec.rb

```diff
@@ -0,0 +1,79 @@
+require './lib/game.rb'
+require './lib/frame.rb'
+require './lib/final_frame.rb'
+require './lib/roll.rb'
+require 'capybara/rspec'
+
+feature 'Bowling' do
+  scenario 'gutter game' do
+    game = Game.new
```

Comment:
Perhaps you can use 'before' block before all the scenarios run?

Reply:
agree :D